stdgo/reflect/Reflect.hx

```diff
@@ -194,34 +164,53 @@ enum GoType {
 	GT_float64;
 	GT_complex64;
 	GT_complex128;
-	GT_array(elem:GoType, len:GoInt);
-	GT_chan(elem:GoType, dir:GoInt);
-	GT_func(input:Array<GoType>, output:Array<GoType>);
+	GT_array(elem:Type, len:GoInt);
+	GT_chan(elem:Type);
+	GT_func(input:Array<Type>, output:Array<Type>);
 	GT_interface(haxeClassPath:GoString);
-	GT_map(key:GoType, value:GoType);
-	GT_ptr(elem:GoType);
-	GT_slice(elem:GoType);
+	GT_map(key:Type, value:Type);
+	GT_ptr(elem:Type);
+	GT_slice(elem:Type);
 	GT_string;
 	GT_struct(fields:Array<StructField>);
 	GT_unsafePointer;
 	GT_namedType(haxeClassPath:GoString);
 }
 
+@:using(stdgo.reflect.Reflect._Type__extension)
+abstract Type(GT_enum) from(GT_enum) to(GT_enum) {
```

Comment:
Doesn't https://api.haxe.org/Type.html interfere with this?

Reply:
I thought that too, but it seems to work ... and Haxe Type function calls are used extensively in this code.

From other modules this Type would presumably be named stdgo.reflect.Reflect.Type anyway, which would avoid confusion.